src/OrbitGl/App.cpp

```diff
@@ -1218,56 +1209,30 @@ void OrbitApp::LoadSymbols(const std::filesystem::path& symbols_path, ModuleData
   });
 }
 
-ErrorMessageOr<void> OrbitApp::GetFunctionInfosFromHashes(
-    const ModuleData* module, const std::vector<uint64_t>& function_hashes,
-    std::vector<const FunctionInfo*>* function_infos) {
-  CHECK(module != nullptr);
-  const ProcessData* process = GetTargetProcess();
-  if (process == nullptr) {
-    return ErrorMessage(absl::StrFormat(
-        "Unable to get function infos for module \"%s\", because no process is selected",
-        module->file_path()));
-  }
-  if (!process->IsModuleLoaded(module->file_path())) {
-    return ErrorMessage(absl::StrFormat(
-        R"(Unable to get function infos for module "%s", because the module is not loaded by process "%s")",
-        module->file_path(), process->name()));
-  }
-
-  size_t count_missing = 0;
-  for (const uint64_t function_hash : function_hashes) {
-    const FunctionInfo* function = module->FindFunctionFromHash(function_hash);
-    if (function == nullptr) {
-      count_missing++;
+void OrbitApp::SelectFunctionsFromHashes(const ModuleData* module,
+                                         const std::vector<uint64_t>& function_hashes) {
+  for (const auto function_hash : function_hashes) {
+    const orbit_client_protos::FunctionInfo* const function_info =
+        module->FindFunctionFromHash(function_hash);
+    if (function_info == nullptr) {
+      ERROR("Could not find function hash %#x in module %s", function_hash, module->file_path());
```

Comment:
@dpallotti I changed the error message as suggested in the other PR. You also suggested to use the `ll` length modifier (`%llx`). That's not necessary with abseil.  Quote from the docs:

```
A length modifer to modify the length of the data type. In StrFormat(), these values are ignored (and not needed, since StrFormat() is type-safe) but are allowed for backwards compatibility
```

Reply:
Ah race condition.

Reply:
Ah nice, thanks for explaining!